### ISSUE TITLE
Update Tagalog translations

### ISSUE THREAD

**Issue by diamant3** [squidfunk/mkdocs-material#6684]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "tl",
  "action.edit": "I-edit ang pahinang ito",
  "action.skip": "I-skip tungo sa nilalaman",
  "action.view": "Tingnan ang pinagmulan ng pahinang ito",
  "announce.dismiss": "Huwag mo nang ipakita muli ito",
  "blog.archive": "Archive ⬅️",
  "blog.categories": "Kategorya",
  "blog.categories.in": "sa",
  "blog.continue": "Magpatuloy sa pagbabasa",
  "blog.draft": "Draft ⬅️",
  "blog.index": "Bumalik sa index",
  "blog.meta": "Metadata ⬅️",
  "blog.references": "Mga kaugnay na link",
  "clipboard.copy": "Kopyahin sa clipboard",
  "clipboard.copied": "Nakopya mula sa clipboard",
  "consent.accept": "Tanggapin",
  "consent.manage": "Pamahalaan ang mga setting",
  "consent.reject": "Tanggihan",
  "footer": "Lagdang Pangwakas",
  "footer.next": "Susunod",
  "footer.previous": "Nakaraan",
  "header": "Pamuhatan",
  "meta.comments": "Mga Komento",
  "meta.source": "Pinagmulan",
  "nav": "Nabigasyon",
  "readtime.one": "1 minutong basahin",
  "readtime.other": "# minutong basahin",
  "rss.created": "RSS feed ⬅️",
  "rss.updated": "RSS feed ng na-update na nilalaman",
  "search": "Hanapin",
  "search.placeholder": "Hanapin",
  "search.share": "Ibahagi",
  "search.reset": "Tanggalin",
  "search.result.initializer": "Sinisimulan ang paghahanap",
  "search.result.placeholder": "Mag-type upang simulan ang paghahanap",
  "search.result.none": "Walang nahanap na dokumento",
  "search.result.one": "1 magkatugmang dokumento",
  "search.result.other": "# magkatugmang mga dokumento",
  "search.result.more.one": "1 meron sa pahina na ito",
  "search.result.more.other": "# meron sa pahina na ito",
  "search.result.term.missing": "Nawawala",
  "select.language": "Pumili ng lenguwahe",
  "select.version": "Pumili ng bersyon",
  "source": "Pumunta sa repository",
  "source.file.contributors": "Mga nag-ambag",
  "source.file.date.created": "Nagawa",
  "source.file.date.updated": "Huling update",
  "tabs": "Mga tala",
  "toc": "Talaan ng nilalaman",
  "top": "Bumalik sa taas"
}[key] }}{% endmacro %}
```

### Country flag

:flag_ph:

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to the best of my knowledge.
- [ ] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by diamant3** [comment]
I leave others because of the jargon. thanks!

**Comment by squidfunk** [comment]
Awesome, thanks! Added in 6128ab92f.

**Comment by diamant3** [comment]
Thanks you as well!

**Comment by squidfunk** [comment]
Keeping open until released.

**Comment by diamant3** [comment]
Sorry, got it! thanks again!

**Comment by squidfunk** [comment]
Released as part of [9.5.5](https://squidfunk.github.io/mkdocs-material/changelog/#9.5.5).
